procedures/MT/rotator/rotator_analysis.ipynb

```diff
@@ -0,0 +1,914 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def analyze_tracking_target(data, time_operation, time_start, time_end, show_figure=True):
</pre><p>This is almost exactly the same name as the previous method and has exactly the same definition. it took me awhile to figure this out! Can you change one of the names and descriptions?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Renamed in the commit: [3f0a032](https://github.com/lsst-ts/ts_notebooks/pull/96/commits/3f0a032629e2db3344107ff9821c35efe5e44afd).

But I should say they are just the single and plural forms.

Thanks!